app/policies/proposal_policy.rb

```diff
@@ -98,7 +98,7 @@ def pending_delegate?
 
   def pending_approval!
     check(self.pending_approver? || self.pending_delegate?,
-          "A response has already been logged a response for this proposal")
+          "A response has already been logged for this proposal")
```

Comment:
I was imagining waiting to hear via the support queue, which is where we learned about this original problem (the authorization error on multiple approval attempts).